lib/assert.js

```diff
@@ -122,8 +122,16 @@ function wrapAssertions(callbacks) {
 			}
 		},
 
-		log(text) {
-			log(this, text);
+		log() {
+			const args = Array.prototype.slice.call(arguments);
+			args.forEach((value, index) => {
+				args[index] = typeof value === 'string' ?
+					value : concordance.format(value, concordanceOptions);
+			});
```

Comment:
Since you're really mapping `args` anyway this would be better as:

```js
const args = Array.from(arguments, value => {
  // ...
});
```

Reply:
that makes sense! I was trying to find a better way :)